pkgs/applications/video/mpv/wrapper.nix

```diff
@@ -33,6 +34,9 @@ let
     ] ++ lib.optionals mpv.vapoursynthSupport [
       mpv.vapoursynth.python3
     ]);
+
+    availableFontPaths = map (x: "${x}/share/fonts") (builtins.filter (n: builtins.pathExists "${n}/share/fonts") scripts);
```

Comment:
Hmm I think this causes "Import from derivation" (IFD) something which Hydra doesn't allow running. It happens when you evaluate a Nix variable, based on the contents of a derivation - meaning you have to build the derivation in order to evaluate the variable - something that the engineers of Hydra didn't want it to experience it. 

However, I think there are other places where this happens in this file where this happens, and most importantly, CI is green. So I guess this is OK.

Reply:
> My afterthoughts are multiple OSCs aren't common, and we can always do this in a new PR if wanted :)

Oh Actually I didn't want you to revert this. I really meant what I wrote at the end that "I guess this is OK". I just wanted you to acknowledge that what you were doing is a bit fishy, and wanted to share that thought.